File: test/fixtures/javascript-jquery/get_referer.js
$.ajax({
  url: 'http://localhost:28139',
  crossDomain: true,
  headers: {
    'X-Requested-With': 'XMLHttpRequest',
    'User-Agent': 'SimCity',
    'Referer': 'https://website.com'
  }
}).done(function(response) {
  console.log(response);
});

File: test/fixtures/ruby-httparty/get_referer.rb
require 'httparty'

url = 'http://localhost:28139'
headers = {
  'X-Requested-With': 'XMLHttpRequest',
  'User-Agent': 'SimCity',
  'Referer': 'https://website.com',
}
res = HTTParty.get(url, headers: headers)
